mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessageConverter.java

```
@@ -79,6 +86,7 @@ DeletedMessage convert(DeletedMessageMetadata deletedMessageMetadata, MailboxMes
         try {
             return Optional.ofNullable(messageBuilder.parseMessage(mailboxMessage.getFullContent()));
         } catch (IOException e) {
+            LOGGER.warn("Can not parse the message {} cause of {}", mailboxMessage.getUid(), e.getMessage());
```

Why do we swallow the exception stacktrace?